packages/visx-shape/test/Arc.test.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import mockConsole from 'jest-mock-console';
-import { shallow, mount } from 'enzyme';
-
+import { render } from '@testing-library/react';
 import { Arc } from '../src';
 import { ArcProps } from '../src/shapes/Arc';
+import '@testing-library/jest-dom';
```

Comment:
not sure if this will throw a lint error for being after local imports 🤔  fine if it doesn't throw!